lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3340,19 +3340,13 @@ swift::ASTContext *SwiftASTContext::GetASTContext() {
              moduleCachePath.c_str());
 
   // Compute the prebuilt module cache path to use:
-  // <resource-dir>/<platform>/prebuilt-modules
+  // <resource-dir>/<platform>/prebuilt-modules/<version>
   llvm::Triple triple(GetTriple());
-  llvm::SmallString<128> prebuiltModuleCachePath(GetResourceDir(triple));
-  StringRef platform;
-  if (swift::tripleIsMacCatalystEnvironment(triple)) {
-    // The prebuilt cache for macCatalyst is the same as the one for macOS,
-    // not iOS or a separate location of its own.
-    platform = "macosx";
-  } else {
-    platform = swift::getPlatformNameForTriple(triple);
-  }
-  llvm::sys::path::append(prebuiltModuleCachePath, platform,
-                          "prebuilt-modules");
+  unsigned Maj, Min, Micro;
+  triple.getOSVersion(Maj, Min, Micro);
```

Comment:
Is this getting the version directly out of the target triple (i.e. the minimum deployment target version)?  The prebuilt module path uses the SDK version.  In Swift we are doing something like this on Darwin

* Use `clang::driver::parseDarwinSDKInfo` which gets a `DarwinSDKInfo`
* Use `swift::getTargetSDKVersion` to interpret it into a correct version tuple for swift.

Reply:
I'll see if I can again reuse the Swift code.

Reply:
The new code looks less nice, but it replicates what the Swift frontend does. I would have liked to just use LangOpts.SDKVersion instead of doing this manually, but that isn't actually populated by what LLDB does.